english/bite-the-marshmallow/tasty-marshmallow.md

```diff
@@ -7,7 +7,7 @@ thought marshmallows, so white, so innocent, could cause such
 excruciating pain! But come to think of it, you've never gone
 to the dentist about that root canal he recommended almost a
 year ago, and perhaps you should have gone back instead of
-trying to save a few bucks...
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.